kubernetes-ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1

```

Comment:
You've got a mix of the Kubernetes provider and YAML - I'd pick one and stick with it for consistency.  There are perks to both but I'll let you decide how you want to structure it.